website/docusaurus.config.js

```diff
@@ -61,6 +61,16 @@ var siteSettings = {
       appId: ALGOLIA_APP_ID ? ALGOLIA_APP_ID : 'dbt'
       //debug: true,
     },
+    announcementBar: {
+      id: "live_qa",
```

Comment:
Is there any significance to the name you chose for the `id`?

Reply:
Negative, the docs state `Any value that will identify this message.`
https://docusaurus.io/docs/api/themes/configuration#announcement-bar-id